src/app/static/src/app/components/plots/bubble/Bubble.vue

```diff
@@ -63,10 +63,10 @@ const getSizeIndicator = () => {
 const colourIndicator = ref<string>(getColourIndicator());
 const sizeIndicator = ref<string>(getSizeIndicator());
 const colourIndicatorMetadata = computed<ChoroplethIndicatorMetadata>(() => {
-    return getIndicatorMetadata(store, colourIndicator.value)
+    return getIndicatorMetadata(store, "bubble", colourIndicator.value)
```

Comment:
would probably extract these out into a top level variable called `plotName` or something like that (i have done that in my branch of the input choropleth too) just so its in one variable although no biggy